jobs/build/olm_bundle/olm_bundles.groovy

```diff
@@ -0,0 +1,69 @@
+buildlib = load('pipeline-scripts/buildlib.groovy')
+commonlib = buildlib.commonlib
+
+/*
+ * Temporary wrapper to use my fork of doozer until
+ * https://github.com/openshift/doozer/pull/198 gets merged
+ */
+def doozer(cmd) {
+    commonlib.shell(
+        script:"""
+            python3 /home/jenkins/talessio/doozer/doozer \
```

Comment:
obviously i won't merge it like this :-D

doozer PR is merged but not yet deployed on buildvm.